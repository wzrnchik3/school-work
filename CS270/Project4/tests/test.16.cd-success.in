cd /usr/bin
echo alive in:
pwd
cd /home
echo and now in:
pwd
