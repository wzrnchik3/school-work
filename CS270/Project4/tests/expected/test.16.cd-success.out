alive in:
/usr/bin
and now in:
/home
